rocAL/rocAL/source/loaders/image/image_loader.cpp

```diff
@@ -288,7 +288,7 @@ ImageLoader::update_output_image()
     }
     _output_names = _output_decoded_img_info._image_names;
     _output_image->update_image_roi(_output_decoded_img_info._roi_width, _output_decoded_img_info._roi_height);
-
+    _output_image->update_image_original_roi(_output_decoded_img_info._original_width, _output_decoded_img_info._original_height);
```

Comment:
please remove "roi" from all the fucntions related to original dims

Reply:
Renamed all functions accordingly in the latest commit